lib/htmx_rails/base.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'htmx_rails/version'
+
+module HtmxRails
```

Comment:
should we have HtmxRails or `Htmx::Rails` ? I think the second is more consistent with the `-` meaning. https://guides.rubygems.org/name-your-gem/